Fabric.Authorization.AccessControl/src/app/services/global/initializer.service.ts

```diff
@@ -1,12 +1,14 @@
 import { Injectable, Inject } from '@angular/core';
 import { IAuthService } from './auth.service';
+import { IService, ServicesService } from './services.service';
 
 @Injectable()
 export class InitializerService {
 
-  constructor(@Inject('IAuthService')private authService: IAuthService) { }
+  constructor(@Inject('IAuthService')private authService: IAuthService, private servicesService: ServicesService) { }
 
   initialize() {
-    return this.authService.initialize();
+    return this.authService.initialize()
+      .then(() => this.servicesService.initialize());
```

Comment:
I believe this is causing the AuthGuard to not fire off because the AuthGuard code isn't created until after this initializer service runs.

Looking to see if there's a better place for creating the services.service's list of services.

Reply:
Is there a reason to not simply initialize the services.service in its own constructor, that way it initializes when its needed, instead of eagerly initializing it here? Perhaps with the changes we made to get the identity and access control ui urls from config instead of discovery all the time, that option is now open to us?